module/applications/activity/save-sheet.mjs

```diff
@@ -49,7 +49,7 @@ export default class SaveSheet extends ActivitySheet {
     }));
     context.calculationOptions = [
       { value: "custom", label: game.i18n.localize("DND5E.SAVE.FIELDS.save.dc.CustomFormula") },
```

Comment:
Could we use blank as the value for 'Custom Formula' here like we do with this new Check Activity? Having it be the string 'custom' causes some issues with newly-migrated Save Activities. We could fix the migration to appropriately migrate the `calculation` field to the `"custom"` string, but I think I'd prefer we be consistent here, and I like using blank as the de-facto 'default' option.

Reply:
The reason I use `custom` rather than a blank value for saves is to allow for auto-detecting when a value hasn't been filled in yet and smartly filling it in based on whether the activity was created on a spell or not (spells default to "spellcasting" while other items default to "custom"). I suppose we could set some other arbitrary value to allow us to check for an unset initial value.

Reply:
I was mainly hoping to easily fix this issue whereby the custom formula input is incorrectly disabled when you first create a save activity (on a non-spell I suppose).

https://github.com/user-attachments/assets/f4e46066-5bfb-4a6d-a752-c70df757b6a0

Reply:
Huh, that was very specifically set up to prevent that issue. Looks like that is because the template is checking `source.save.dc.calculation` rather than the derived value.